src/results/resultTable.ts

```diff
@@ -0,0 +1,385 @@
+/**
+ * Copyright 2021 RelationalAI, Inc.
```

Comment:
It's 2022!

Reply:
Yeah, probably a separate PR with a mass change for all files.